docs/cow-protocol/tutorials/arbitrate/README.mdx

```diff
@@ -53,6 +53,39 @@ They receive the pre-processed auction instance from the driver sub-component an
 
 The main flow of arbitrating an auction end-to-end can be seen in the following sequence diagram:
 
+```mermaid
+sequenceDiagram
+    actor User
+    participant orderbook
+    participant autopilot
+    participant solver
+
+    User->>orderbook: /post_order
+    autopilot->>orderbook: Request solvable orders
+    activate autopilot
+    activate orderbook
+    orderbook->>autopilot: Response solvable orders
```

Comment:
Should we also show the DB here since the autopilot is actually just talking to that instead of the orderbook or would that go into too much detail?
Not sure what level of detail we want to have here.

Reply:
Since the text is talking about three main components, I actually like that the diagram focusses on that. Will move the diagram up to the architecture section.